openhands/app_server/sandbox/sandbox_spec_service.py

```diff
@@ -60,9 +61,35 @@ class SandboxSpecServiceInjector(
     pass
 
 
-def get_default_agent_server_image():
+def get_default_agent_server_image() -> str:
     agent_server_image_repository = os.getenv('AGENT_SERVER_IMAGE_REPOSITORY')
     agent_server_image_tag = os.getenv('AGENT_SERVER_IMAGE_TAG')
     if agent_server_image_repository and agent_server_image_tag:
         return f'{agent_server_image_repository}:{agent_server_image_tag}'
     return AGENT_SERVER_IMAGE
+
+
+def get_default_agent_server_env() -> dict[str, str]:
```

Comment:
```suggestion
def get_agent_server_env() -> dict[str, str]:
```
Nit: It seems the purpose of this function is precisely to allow a custom agent server env, not only defaults. 
(same goes for get_default_server_image() actually)

Reply:
Renamed!